lib/cfer/cfn/client.rb

```diff
@@ -59,19 +59,25 @@ def converge(stack, options = {})
         }
       end
 
-      options = {
+      ## arguments to be passed to {create,update}_stack
+      stack_options = {
         stack_name: name,
         template_body: stack.to_cfn,
         parameters: parameters,
         capabilities: response.capabilities
       }
 
+      ## optional policy can be applied to both create and update
+      stack_options.merge!(parse_stack_policy(:stack_policy, options[:stack_policy]))
```

Comment:
The tests for the stack stuff uses [rspec-mocks](https://github.com/rspec/rspec-mocks) to mock the AWS API, and look for a specific options hash to be sent to the SDK.

In the stack creation test, the `options` hash is passed in here: https://github.com/seanedwards/cfer/blob/develop/spec/cfn_spec.rb#L55

So if you added a `stack_policy: '{}'` argument to the `converge` call, you should see a `stack_policy_body: '{}'` added to the SDK options here: https://github.com/seanedwards/cfer/blob/develop/spec/cfn_spec.rb#L43

Similarly, the test will check the inputs to the AWS SDK for updates here: https://github.com/seanedwards/cfer/blob/develop/spec/cfn_spec.rb#L98

So you should be able to add `stack_policy: '{}'` and `stack_policy_during_update: '{}'` to the options hash here: https://github.com/seanedwards/cfer/blob/develop/spec/cfn_spec.rb#L118 and see it show up in the stack options.